### PR TITLE
edit DG UG and parser

### PR DESCRIPTION
**Pull request by JeremyTakigawa**

### CODE REVIEW

File: src/main/java/seedu/parser/Parser.java [hunk 1]
```diff
@@ -55,53 +56,59 @@ public static Command parse(String userInput) throws EscException {
...
+            case HelpCommand.COMMAND_WORD:
+                return new HelpCommand();
 
-        default:
-            throw new EscException(INCORRECT_COMMAND + HelpCommand.MESSAGE_USAGE);
+            case ExitCommand.COMMAND_WORD:
+                return new ExitCommand();
+
+            default:
+                throw new EscException(INCORRECT_COMMAND + HelpCommand.MESSAGE_USAGE);
```
Comment (kcubey) [inline src/main/java/seedu/parser/Parser.java]: According to the convention set in lecture, these should not have the extra tab spacing. The default & set cases should be in line with the switch command. Pls update!

(From the textbook)
image

File: src/main/java/seedu/parser/Parser.java [hunk 2]
```diff
@@ -367,7 +392,7 @@ private static LocalDate getEventDate(String argument) throws EscException {
         DateTimeFormatter dateKey = DateTimeFormatter.ofPattern("[dd/MM/yyyy][d/M/yyyy][dd/MM/yy][d/M/yy]"
                 + "[yyyy/MM/dd][yyyy-MM-dd][yyyy-M-d]"
                 + "[dd-MM-yyyy][d-M-yyyy][dd-MM-yy][d-M-yy]"
-                + "[dd.MM.yy][d.M.yy][dd.MM.yyyy][d.M.yyyy][yyyy.MM.dd]"
+                + "[dd.MM.yy][d.M.yy][dd.MM.yyyy][d.M.yyyy]"
```
Comment (kcubey) [inline src/main/java/seedu/parser/Parser.java:395]: Is the yyyy.MM.dd date format not working? That's why it's removed?

@JeremiasLiew please do take note and update the code/UG/DG as necessary

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -310,27 +310,22 @@ The product aims to provide students with more convenient way of doing revision.
...
 
-Method 1: Using IntelliJ JUnit test runner
+Method : Using IntelliJ JUnit test runner
     
-    To run all tests, right-click on the src/test/java folder and choose Run 'All Tests'
-
-Method 2: Using Gradle
-
-    Open a console and run the command gradlew clean test (Mac/Linux: ./gradlew clean test)
-See UsingGradle.adoc for more info on how to run tests using Gradle.
```
Comment (kcubey) [inline docs/DeveloperGuide.md:333]: Why's this part removed? Is it not relevant? 🤔

**Review by kcubey — CHANGES_REQUESTED** [review]
Please remove the data folder! Otherwise may cause merge conflicts

File: src/main/java/seedu/parser/Parser.java (hunk 2 above)
Comment (JeremyTakigawa) [reply src/main/java/seedu/parser/Parser.java:395]: I didn't change this part, should be someone else

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (JeremyTakigawa) [reply docs/DeveloperGuide.md:333]: I couldn't get it running on mine so I removed it in case our app is not configured properly

**Pull request closed without merging**